api/src/paths/telemetry/manual/deployments.ts

```diff
@@ -0,0 +1,64 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { manual_telemetry_responses } from '.';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { BctwService, getBctwUser } from '../../../services/bctw-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/telemetry/manual');
+
+export const POST: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          discriminator: 'SystemUser'
```

Comment:
Just checking: A bunch of endpoints in this PR specify only SystemUser level security. Is that correct?